specs/Renting/P2PCarRental/doc/spec.md

```diff
@@ -0,0 +1,75 @@
+# P2PCarRental
+
+## Description
+
+This entity contains a description of a peer to peer rental car.
+It aims to describe basic characteristics of the vehicle and it's owner. In addition to this it has some other details, like price and availability.
+We hope that this model receives contributions and will be developed further by the community.
+
+This data model has been created in co-operation with share it blox car.
+
+## Data Model
+
+Please see also the [example.json](../example.json) and
+[schema.json](../schema.json). Look at those if the explanations in this
+document don't open up the usage.
+
+A JSON Schema corresponding to this data model can be found
+[here](../schema.json). All the following entries are mandatory.
+
+-   `id` : Unique identifier.
+
+-   `type` : Entity type. It must be equal to `P2PCarRental`.
+
+-   `vehicle_model` : The model of the vehicle omitting make, for example "159"
```

Comment:
please follow the general data model guidelines ... and the documentation template